src/triage/component/catwalk/utils.py

```diff
@@ -202,3 +203,37 @@ def save_db_objects(db_engine, db_objects):
             )
         f.seek(0)
         postgres_copy.copy_from(f, type(db_objects[0]), db_engine, format="csv")
+
+
+def class_path_implements_parallelization(class_path):
```

Comment:
I was wondering if we can change this behavior. I think you can avoid the `n_job` problem by only targeting the `catwalk.estimators.classifiers.ScaledLogisticRegression` instead of the `LogisticRegression` (actually we should strongly advice not to use the later). 

If you remove the `n_job` argument from the SLR, I guess your trick will work.

Reply:
What @saleiro thinks about this?

Reply:
🤔 ☝️

Reply:
I would refrain from hardcding the large ones. We could check if the user passed a 'n_jobs' argument in the configuration parameters for the target class_path. This way this functionlity would be equally compatible with xgboost or lightgbm cause they use the 'n_jobs' parameter too.